docs/input/index.html

```diff
@@ -124,16 +123,17 @@ <h3 class="title">Code Snippets</h3>
-              style="width: 770px; height: 890px; border: 0; transform: scale(1); overflow: hidden"
+              style="order: 0; transform: scale(1); overflow: hidden"
```

Comment:
try not to use inline classes. It will be better as I can see it is adding redundancy to your code.